types/controller.go

```diff
@@ -144,3 +150,10 @@ func synchronizeLookups(ticker *time.Ticker,
 func (c *Controller) Topics() []string {
 	return c.TopicMap.Topics()
 }
+
+func checkAsyncRoute(config *ControllerConfig) string {
```

Comment:
I find the name `checkAsyncRoute` a little unclear. Would you mind calling it `gatewayRoute` or something similar?

Reply:
Sound fair will do 👍

Reply:
Changed 👍